scripts/install_backend_python_libs.py

```diff
@@ -401,6 +401,11 @@ def _get_possible_normalized_metadata_directory_names(
         normalize_directory_name(
             '%s-%s.egg-info' % (
                 library_name.replace('-', '_'), version_string)),
+        normalize_directory_name(
+            '%s-%s-py2.7.egg-info' % (library_name, version_string)),
```

Comment:
Can you please add a code-comment on top explaining why we have this py2.7 here? (Around line 392)

Reply:
Done.